src/Template/Layout/js/app/components/relation-view/relation-view.js

```diff
@@ -86,6 +82,9 @@ export default {
     },
 
     computed: {
+        listView() {
+            return this.relationName != 'poster' && this.relationName != 'attach';
```

Comment:
instead of using relation names that are dynamic we could define this behavior looking for `uploadable` media types in related types 

See: https://github.com/bedita/manager/blob/master/src/Template/Element/Form/relation.twig#L78 

Here we decide to show a drop area, but the logic is the same. We could probably define a Twig boolean variable and pass it as a Vue component `prop`